indica/operators/extrapolate_impurity_density.py

```diff
@@ -891,17 +947,29 @@ def objective_func(objective_array: Sequence, time: float):
                     lower_amp_bound,
                     lower_width_bound,
                     lower_pos_bound,
+                    0.1 * asymmetry_parameter.min(),
                 ]
             ),
             np.array(
                 [
                     upper_amp_bound,
                     upper_width_bound,
                     upper_pos_bound,
+                    10 * asymmetry_parameter.max(),
```

Comment:
@elitherl Do these bounds seem good? Maybe we should subtract something from the lower bound in case it is small and positive to allow it to be negative?

Reply:
Could base the bounds on some factor (e.g. 5, 10) * the absolute value of the maximum, to account for negative asymmetry

Reply:
This seems like a good idea but I am now struggling to get the optimizer to converge for isel(t=1).

Reply:
Got it to converge by tweaking `xtol` up a bit, hopefully the changes to `x_scale` should make `xtol` better behaved.